Rakefile

```diff
@@ -14,11 +14,32 @@ Rake::TestTask.new do |t|
   t.pattern = "spec/*_spec.rb"
 end
 
+DATA_DEST_PATH   = File.expand_path('../source/data/awards.json', __FILE__)
+DATA_SRC_PATH    = File.expand_path('../data/awards.json', __FILE__)
+INJECT_DEST_PATH = File.expand_path('../data/injectable.yml', __FILE__)
+
+desc "Default task is to ensure injectable data is available"
+task :default => [INJECT_DEST_PATH]
+
 desc 'reprocess data to remove bad images, add hyphens'
-task :reprocess do
+file DATA_DEST_PATH do
   require 'bundler/setup'
-  source = File.expand_path('../data/awards.json', __FILE__)
-  destination = File.expand_path('../source/data/awards.json', __FILE__)
-  processor = Reprocess.new(source, destination)
+  puts "Reprocessing source data..."
+  processor = Reprocess.new(DATA_SRC_PATH, DATA_DEST_PATH)
   processor.reprocess
+  puts "Done"
+end
+
+desc 'Add first awards to HTML for faster load'
+file INJECT_DEST_PATH => DATA_DEST_PATH do
```

Comment:
Cool, I didn't know about the file tasks! Just looked them up and they seem like the perfect choice in this case!